x/concentrated-liquidity/types/migration.go

```diff
@@ -8,3 +8,750 @@ var MigratedIncentiveAccumulatorPoolIDs = map[uint64]struct{}{
 	1292: {},
 	1431: {},
 }
+
+var FinalIncentiveAccumulatorPoolIDsToMigrated = map[uint64]struct{}{
```

Comment:
Mentioned in another comment but I think this should live in the upgrade folder.

Reply:
Fixed in 63ea98f2121e7fd321c348acc0bc65ccb5a69df3

Reply:
Do we want to remove this now that a copy exists in the upgrade folder?

Reply:
Moved in the previous commit, must have missed the delete. Ty :bow: 

Moved both of the pool id maps to their relevant upgrade folders => a410aa3c45d5808c8d602a48df951aeb5e6cfe24